hardware/is_camera/src/camera.cc

```diff
@@ -230,6 +233,9 @@ namespace is_camera {
         bayer_img_msg_buffer_[i]->data.resize(kImageWidth * kImageHeight);
       }
     }
+    // Start bayer enable/disable service
+    enable_bayer_srv_ = nh->advertiseService(SERVICE_HARDWARE_BAYER_ENABLE,
```

Comment:
I'm curious about the intended new conops for managing bayer publishing.

The old approach was: If disabled in the config, no bayer publishing. If enabled in the config, publish bayer only if there is a subscriber that wants it. (So there was already a way to turn publishing on/off dynamically by changing recording profiles or whatever, assuming you can control the subscribers.)

What is the intended use case for making dynamic enabe/disable calls to this service? I guess I want to avoid the usability pitfall of setting it to be disabled by default so that users need to dynamically enable it when needed, but they might not realize enabling the bayer images in the image sampler is not enough, and have no good way to discover this extra step.

Reply:
This is not an extra step, but an OR possibility, please note that if the config is set to enable, no extra action is needed.
In case the fsw was started with bayer disable there is no way of enabling it without having to restart it and manually changing the file. We've had similar issues is ops with things like the arm enable, where if we forgot to do it beforehand, we had to restart the entire sw stack (which we fixed already). This might not be super important for ISAAC activities because they always use bayer recording, but for things like crew minimals and lab testing it will make it much easier.